### ISSUE TITLE
Feature Request: Render True Tile Indicator Behind Player Model

### ISSUE THREAD

**Issue by chris023** [vincent0955/Visual-metronome#38]
Title; Currently the true tile metronome indicator renders in front of the player model.  Add a configurable option (or make it the default) to render it behind the player model.

**Comment by chris023** [comment]
"Improved Tile Indicators" in conjunction with this plugin results in this behavior, for anyone else who ends up here.

**Comment by vincent0955** [comment]
Yeah I see you've figured it out yourself, Improved Tile Indicators' render under model option does indeed work with this plugin's tile indicator now.
